fla/layers/mom_varlen.py

```
@@ -0,0 +1,724 @@
+
+
+
+if TYPE_CHECKING:
+    from transformers.processing_utils import Unpack
+
+    from fla.models.utils import Cache
+
+from transformers.utils import is_flash_attn_2_available
+if is_flash_attn_2_available():
+    from flash_attn.bert_padding import index_first_axis, pad_input, unpad_input 
+    from flash_attn.bert_padding import pad_input
```

Ruff (0.11.9)

Redefinition of unused `pad_input` from line 28

Remove definition: `pad_input`

(F811)